DigitalOcean.API/Models/Requests/LoadBalancer.cs

```diff
@@ -43,30 +40,32 @@ public class LoadBalancer {
 		/// An object specifying sticky sessions settings for the Load Balancer.
 		/// </summary>
 		[JsonProperty("sticky_sessions")]
-		public StickySessions StickySessions { get; set; }
+		public Models.Responses.StickySessions StickySessions { get; set; }
```

Comment:
I feel like `StickySessions` should have a presence under the `Request` namespace here.

Reply:
Done